openapi.yaml

```diff
@@ -15651,6 +15918,85 @@ components:
           description: When this Domain Record was last updated.
           example: '2018-01-01T00:01:01'
           readOnly: true
+    EntityTransfer:
+      type: object
+      description: >
+        An object representing an Entity Transfer.
+      properties:
+        token:
+          x-linode-cli-display: 1
+          type: string
+          format: uuid
+          description: >
+            The token used to identify and accept this transfer.
+          example: "123E4567-E89B-12D3-A456-426614174000"
+        status:
+          x-linode-filterable: true
+          type: string
+          enum:
+          - accepted
+          - cancelled
+          - completed
+          - failed
+          - pending
+          - stale
+          description: >
+            The status of the transfer request.
+
+
+            `accepted`: The transfer has been accepted by another user and is currently in progress. Transfers can take several hours to complete.
+
+
+            `canceled`: The transfer has been canceled by the sender.
```

Comment:
we are using the double L "cancelled" for the enum. You have it correct in the list above, but the two places here in this sentence should be changed.